modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -353,7 +353,7 @@ package object engine {
 
   def pure[A](a: A): HandleP[A] = Applicative[HandleP].pure(a)
 
-  private val unit: HandleP[Unit] = pure(Unit)
+  private val unit: HandleP[Unit] = pure(())
```

Comment:
I'm still confused about when to use `()` and `Unit` in Scala. Is `Unit` supposed to be used only at the type level and `()` at the value level? Is it a stylistic thing or is there more to it?